integration-tests/tests/src/tests/array-buffer.ts

```diff
@@ -0,0 +1,144 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2021 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+import { expect } from "chai";
+import { Realm } from "realm";
+
+const isNodeProcess = typeof process === "object" && process + "" === "[object process]";
+
+const SingleSchema: Realm.ObjectSchema = {
+  name: "PrimitiveData",
+  properties: {
+    a: "data",
+  },
+};
+
+describe("ArrayBuffer", () => {
+  let realm: Realm;
+  beforeEach(() => {
+    Realm.clearTestState();
+    realm = new Realm({ schema: [SingleSchema] });
+  });
+
+  afterEach(() => {
+    realm.close();
+  });
```

Comment:
Same as your other PR - I'd like for us to use the `openRealmBeforeEach` hook here.